File: 2024/go/utils/output_utils.go
package utils

import (
	"fmt"
)

func PrintOutput(val string, useTestData bool) {
	var outputString = "Total [using puzzle data]: "

	if useTestData {
		outputString = "Total [using test data]: "
	}

	fmt.Println(outputString, val, "\n\n")
}

File: 2024/rust/src/utils/output_utils.rs
use std::fs;


pub fn print_output(total: &str, use_test_data: bool) {
	let output_string = if use_test_data {
		"Total [using test data]"
	} else {
		"Total [using puzzle data]"
	};

	println!("{}: {}", output_string, total);
}